pyproject.toml

```
@@ -26,6 +24,15 @@ fsspec = "^2022.7.1"
 termcolor = "^1.1.0"
 gradio = "^3.1.7"
 colorama = ">=0.4.5,<1.0.0"
+jiwer = "^2.5.1"
+transformers = "^4.23.0"
+soundfile = "^0.11.0"
+evaluate = "^0.3.0"
+pyctcdecode = "^0.4.0"
+pypi-kenlm = "^0.1.20220713"
+librosa = "^0.9.2"
+torch = "^1.13.1"
+torchaudio = "^0.13.1"
```

The behaviour of "^" is a bit odd for some reason. For instance, "^1.2.3" means ">=1.2.3, <2.0.0", which is correct, but "^0.1.2" means ">=0.1.2, <0.2.0", so suddenly we're not supporting the next minor version. We can fix this by making these "zero-major versions" more explicit:

```suggestion
soundfile = ">=0.11.0,<1.0.0"
evaluate = ">=0.3.0,<1.0.0"
pyctcdecode = ">=0.4.0,<1.0.0"
pypi-kenlm = ">=0.1.20220713,<1.0.0"
librosa = ">=0.9.2,<1.0.0"
torch = "^1.13.1"
torchaudio = ">=0.13.1,<1.0.0"
```